src/main/java/at/hannibal2/skyhanni/test/DebugCommand.kt

```diff
@@ -145,6 +146,9 @@ object DebugCommand {
             add("skyBlockArea:")
             add("  scoreboard: '${LorenzUtils.skyBlockArea}'")
             add("  graph network: '${IslandAreas.currentAreaName}'")
+            MinecraftCompat.localPlayer.position.let {
+                add("  player pos: ${it.x}, ${it.y}, ${it.z}")
```

Comment:
please use this: `with(location.roundTo(1)) { "/shtestwaypoint $x $y $z pathfind" }`